.circleci/config.yml

```diff
@@ -111,17 +111,24 @@ commands:
 jobs:
   build-and-unit-test:
     docker:
-      - image: cimg/node:15.8.0
+      - image: cimg/node:16.15.0
```

Comment:
getting a standard package-lock.json worked with 16.15.0 and not 15.8.0 (build passing and cache working)